build/charts/theia/values.yaml

```diff
@@ -0,0 +1,103 @@
+clickhouse:
+  # -- Container image to use for ClickHouse.
+  image: 
+    repository: "projects.registry.vmware.com/antrea/theia-clickhouse-server"
+    pullPolicy: "IfNotPresent"
+    tag: "21.11"
+  monitor:
+    # -- Determine whether to run a monitor to periodically check the ClickHouse
+    # memory usage and clean data.
+    enable: true
+    # -- The storage percentage at which the monitor starts to delete old records.
+    # Vary from 0 to 1.
+    threshold: 0.5
+    # -- The percentage of records in ClickHouse that will be deleted when the
+    # storage grows above threshold. Vary from 0 to 1.
+    deletePercentage: 0.5
+    # -- The time interval between two round of monitoring. Can be a plain integer using
+    # one of these unit suffixes ns, us (or µs), ms, s, m, h.
+    execInterval: "1m"
+    # -- The number of rounds for the monitor to stop after a deletion to wait for
+    # the ClickHouse MergeTree Engine to release memory.
+    skipRoundsNum: 3
+    # -- Container image to use for the ClickHouse Monitor.
+    image:
+      repository: "projects.registry.vmware.com/antrea/theia-clickhouse-monitor"
+      pullPolicy: "IfNotPresent"
+      tag: "latest"
+  secret:
+    # -- ClickHouse username. It will be stored in a secret.
+    username: "clickhouse_operator"
+    # -- ClickHouse password. It will be stored in a secret.
+    password: "clickhouse_operator_password"
```

Comment:
I assume that these 2 fields will be stored in the same secret, and not 2 separate secrets?
I suggest changing this to:
```
# -- Credentials to connect to ClickHouse. They will be stored in a secret.
connectionSecret:
    username : "clickhouse_operator"
    password: "clickhouse_operator_password"
```